x-pack/plugins/lens/public/editor_frame_service/editor_frame/editor_frame.tsx

```diff
@@ -94,9 +96,24 @@ export function EditorFrame(props: EditorFrameProps) {
     showMemoizedErrorNotification(error);
   }, []);
 
+  const bannerMessages: React.ReactNode[] | undefined = useMemo(() => {
+    if (activeDatasourceId) {
+      return datasourceMap[activeDatasourceId].getDeprecationMessages?.(
+        datasourceStates[activeDatasourceId].state
+      );
+    }
+  }, [activeDatasourceId, datasourceMap, datasourceStates]);
+
   return (
     <RootDragDropProvider>
       <FrameLayout
+        bannerMessages={
+          bannerMessages ? (
+            <ErrorBoundary onError={onError}>
+              <BannerWrapper nodes={bannerMessages} />
```

Comment:
It doesn't look like the banner wrapper is doing anything

Reply:
@flash1293  it was done in manner with other sections. Do you think we should remove this wrapper?

Reply:
I don't see any problems here, semantically this is the correct.